packages/rich-text/src/plugins/EmbeddedEntityInline/index.tsx

```diff
@@ -120,26 +123,18 @@ async function selectEntityAndInsert(
     ...newEntitySelectorConfigFromRichTextField(sdk.field, INLINES.EMBEDDED_ENTRY),
     withCreate: true,
   };
-  const selection = editor.selection;
-
+  const rteSlide = watchCurrentSlide(sdk.navigator);
   const entry = await sdk.dialogs.selectSingleEntry<Entry>(config);
-  focus(editor); // Dialog steals focus from editor, return it.
 
   if (!entry) {
     logAction('cancelCreateEmbedDialog', { nodeType: INLINES.EMBEDDED_ENTRY });
-    return;
+  } else {
+    insertNodes(editor, createInlineEntryNode(entry.sys.id));
```

Comment:
❓ Not sure, but don't we need the setSelection to replace the content with the new inserted Node?

Reply:
ok nvm still works

Reply:
Yes, I also thought about it and tested that and other cases, but I couldn't find anything broken. E.g.

https://user-images.githubusercontent.com/101926/222710048-8ed8a4d9-84ce-4821-b243-c601be589911.mp4

<br/>
Also, see the [discussion here](https://github.com/contentful/field-editors/pull/1353#discussion_r1119819940) where I ask about potential issues.